examples/3d-tiles/app.js

```diff
@@ -116,8 +122,8 @@ export default class App extends PureComponent {
     /* global URL */
     const parsedUrl = new URL(window.location.href);
     const ionAssetId = parsedUrl.searchParams.get('ionAssetId');
-    const ionAccessToken = parsedUrl.searchParams.get('ionAccessToken');
-    if (ionAccessToken) {
```

Comment:
For some tokens, we can auto-detect the asset id. So it should be possible to just provide a token. 

I think the logic needs to be:

```
if (ionAccessToken || ionAssetId) {
  ionAccessToken = ionAccessToken || ION_ACCESS_TOKEN;
  await this. _loadTilesetFromIonAsset(...);
}
```